### PR TITLE
Devtools release notes.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/tools/devtools/release-notes/release-notes-2.34.1-src.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Deep Links tool updates
+
+* Automatically populate a list of Flutter projects from the connected
+IDE. - [#7415](https://github.com/flutter/devtools/pull/7415), [#7431](https://github.com/flutter/devtools/pull/7431)
+
+## Full commit history
+
+To find a complete list of changes in this release, check out the
+[DevTools git log](https://github.com/flutter/devtools/tree/v2.34.0).
```
Comment (kenzieschmoll) [inline src/tools/devtools/release-notes/release-notes-2.34.1-src.md:63]: 2.34.1 here

**Review by kenzieschmoll — COMMENTED** [review]
nit: since DevTools defaults to dark theme, can we use dark theme screenshots?

**Comment by polina-c** [comment]
> nit: since DevTools defaults to dark theme, can we use dark theme screenshots?

I discussed this with @atsansone som time ago. He suggested to use light theme in documentation because it is much easer to consumed by people with impacted vision.

**Comment by kenzieschmoll** [comment]
> > nit: since DevTools defaults to dark theme, can we use dark theme screenshots?
> 
> I discussed this with @atsansone som time ago. He suggested to use light theme in documentation because it is much easer to consumed by people with impacted vision.

I would challenge whether this is the right thing to do, since users view these release notes embedded inside DevTools, where the default is dark theme. So showing light mode screenshots in a dark theme tool (for most our users who don't change the theme to light mode), is inconsistent.

Whatever we do, we should at least make sure all screenshots in a set of release notes follow the same pattern (there are two light mode and one dark mode in this PR).

**Comment by polina-c** [comment]
> > > nit: since DevTools defaults to dark theme, can we use dark theme screenshots?
> > 
> > 
> > I discussed this with @atsansone som time ago. He suggested to use light theme in documentation because it is much easer to consumed by people with impacted vision.
> 
> I would challenge whether this is the right thing to do, since users view these release notes embedded inside DevTools, where the default is dark theme. So showing light mode screenshots in a dark theme tool (for most our users who don't change the theme to light mode), is inconsistent.
> 
> Whatever we do, we should at least make sure all screenshots in a set of release notes follow the same pattern (there are two light mode and one dark mode in this PR).

I will convert dark screen shot to light.

**Review by kenzieschmoll — APPROVED** [review]

File: src/tools/devtools/release-notes/release-notes-2.34.1-src.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Inspector updates
+
+- Fixes an edge case where widgets from other packages could show up in the inspector
+  tree. - [#7353](https://github.com/flutter/devtools/pull/7353)
+
+## Performance updates
+* Add a setting to include CPU samples in the Timeline. - [#7333](https://github.com/flutter/devtools/pull/7333), [#7369](https://github.com/flutter/devtools/pull/7369)
+
+    ![Timeline settings](/tools/devtools/release-notes/images-2.34.1/7369-timeline-settings.png "Timeline settings")
+
```
Comment (kenzieschmoll) [inline src/tools/devtools/release-notes/release-notes-2.34.1-src.md:24]: these need the site.url

**Pull request merged**